Change container to images (#11)
dockersamples/101-tutorial: docs_en/tutorial/persisting-our-data/index.md

## Container Volumes

> With the previous experiment, we saw that each images is effectively read-only. While containers

Reviewer: How about something like this? Thoughts?

```suggestion
With the previous experiment, we saw that each container restarts from scratch from its image. While containers
```
